src/util/hasher.h

```diff
@@ -76,13 +76,13 @@ class SignatureCacheHasher
     }
 };
 
-struct BlockHasher
+/* struct BlockHasher
```

Comment:
what's going on here?

Reply:
BlockHasher was not included in src/util/hasher.h , just the previous one was commented out , i have fixed it now by modifying the src/util/hasher.h as well as src/validation.h , now BlockHasher is contained only in src/util/hasher.h , removed it from src/validation.h.